airflow/contrib/hooks/salesforce_hook.py

```diff
@@ -73,19 +73,23 @@ def get_conn(self):
             )
         return self.conn
 
-    def make_query(self, query):
+    def make_query(self, query, include_deleted=False, **query_params):
```

Comment:
I thought rather about something like this:
```suggestion
    def make_query(self, query, include_deleted=False, query_params=None):
```